tsconfig.json

```diff
@@ -110,9 +109,6 @@
       ],
       "services/*": [
         "main/services/*"
-      ],
-      "integration-test/*": [
-        "integration-test/*"
```

Comment:
Why did this get removed?

Reply:
Because we changed base URL to be `"baseUrl": "src"`. Having this we do not need aliases that points to the same directory because it is already guaranteed by base URL.